clarity/src/vm/mod.rs

```diff
@@ -79,10 +81,92 @@ pub use crate::vm::contexts::MAX_CONTEXT_DEPTH;
 use crate::vm::costs::cost_functions::ClarityCostFunction;
 pub use crate::vm::functions::stx_transfer_consolidated;
 pub use crate::vm::version::ClarityVersion;
+use std::collections::BTreeMap;
 use std::convert::{TryFrom, TryInto};
 
+use self::analysis::ContractAnalysis;
+use self::ast::ContractAST;
+use self::costs::ExecutionCost;
+use self::diagnostic::Diagnostic;
+
 const MAX_CALL_STACK_DEPTH: usize = 64;
 
+#[derive(Debug, Clone)]
+pub struct ParsedContract {
```

Comment:
I'm not sure where any of these new structs are getting used?  Are they here for something upstream?

Reply:
`ExecutionResult` is used by `EvalHook` which is used by any hooks which are implemented.